doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -636,6 +655,26 @@ \subsubsection{Outputs}\label{outputs-9-008}
 
 This is the hybrid ventilation control status, which can have three integer values: 0, 1, and 2. A zero value indicates no hybrid ventilation control, corresponding to zero value defined in the previous output variable. A value of one indicates that natural ventilation is allowed. A value of two denotes that the natural ventilation is not allowed, so all window/door openings are closed.
 
+\paragraph{Hybrid Ventilation Control HVAC System Opration Elapsed Time {[min]}}\label{hybrid-ventilation-control-hvac-system-opration-elapsed-time}
+
+This output is the HVAC system operation elapsed time in the units of minutes since the HVAC system starts.
+
+\paragraph{Hybrid Ventilation Control Natual Ventilation Elapsed Time {[min]}}\label{hybrid-ventilation-control-natual-ventilation-elapsed-time}
+
+This output is the natural ventilation elapsed time in the units of minutes since the natural ventilation starts.
+
+\paragraph{Hybrid Ventilation Operative Temperature {[C]}}\label{hybrid-ventilation-operative-temperature}
+
+This output is the average of Mean Air Temperature and Mean Radiant Temperature in a thermal zone.
+
+\paragraph{Hybrid Ventilation Lower Limit Operative Temperature {[C]}}\label{hybrid-ventilation-lower-limit-operative-temperature}
+
+The output is the lower acceptability limits in degree of C based on user input selection of either 80\% or 90\%.
```

Comment:
@mjwitte The text was revised.

I added an if statement to ensure the output variables will not show if if adaptive control is not active:

			if ( CheckScheduleValue( HybridVentSysAvailMgrData( SysAvailNum ).ControlModeSchedPtr, HybridVentMode_OperT80 ) || CheckScheduleValue( HybridVentSysAvailMgrData( SysAvailNum ).ControlModeSchedPtr, HybridVentMode_OperT90 ) ) {